packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -133,6 +138,10 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
 		[this.opProcessingHelper, this.callbacksHelper] = this.setUpSampledTelemetryHelpers();
 	}
 
+	protected get deltaManager(): IDeltaManager<ISequencedDocumentMessage, IDocumentMessage> {
```

Comment:
added to ease access to internal IDeltaManager type in shared object implementations